tests/test_uds_auxiliary.py

```diff
@@ -317,3 +318,15 @@ def test_Response_wrap(self, uds_raw_aux_inst, mock_uds_config, mocker):
 
         assert resp.is_negative is True
         assert resp.nrc == NegativeResponseCode.CONDITIONS_NOT_CORRECT
+
+    def test_tester_present_sender(self, uds_raw_aux_inst, mocker):
+
+        send_mock = mocker.patch.object(uds_raw_aux_inst, "send_uds_raw")
+
+        with uds_raw_aux_inst.tester_present_sender(0.1):
+            sleep(0.3)
```

Comment:
Are you mocking time.sleep or does the unit test take 0.3s to run?
If it does maybe reduce the timings

Reply:
Problem solved, I mocked it